onset_maproom/environment.yml

```diff
@@ -13,12 +13,14 @@ dependencies:
   - opencv
   - psycopg2
   - rasterio
-  - scipy
```

Comment:
Did you intend to remove this? Seems like it might have happened accidentally when you rebased.

Reply:
No: I added it when I though I was going to use interpolate_na (or something like that) and then realized bafill was more appropriate, and the one requires bottleneck rather then scipy.